docs/sim/agent.rst

```diff
@@ -51,28 +51,36 @@ SMARTS provide some interface types, and the differences between them is shown i
+    |          debug         |           **T**            |                 **T**                  |              **T**              |         **T**          |                   **T**                 |            **T**             |                **T**               |           **T**         |              **T**              |            **T**             |            **T**             |            **T**            |
 
 `max_episode_steps` controls the max running steps allowed for the agent in an episode. The default `None` setting means agents have no such limit.
 You can move `max_episode_steps` control authority to RLlib with their config option `horizon`, but lose the ability to customize
 different max_episode_len for each agent.
 
-`action` controls the agent action type used. There are three `ActionSpaceType`: ActionSpaceType.Continuous, ActionSpaceType.Lane 
-and ActionSpaceType.ActuatorDynamic.
+`action` controls the agent action type used. There are multiple `ActionSpaceType` options: `ActionSpaceType.Continuous`, `ActionSpaceType.Lane`, `ActionSpaceType.LaneWithContinuousSpeed` 
+`ActionSpaceType.ActuatorDynamic`, `ActionSpaceType.Trajectory`, `ActionSpaceType.TrajectoryWithTime`, `ActionSpaceType.MPC`, `ActionSpaceType.MultiTargetPose`, and `ActionSpaceType.Imitation`.
+
+- `ActionSpaceType.Continuous`: `(float, float, float)` continuous action space with throttle, brake, absolute steering angle. 
+- `ActionSpaceType.ActuatorDynamic`: `(float, float float)` continuous action space with throttle, brake, steering rate. Steering rate means the amount of steering angle change *per second* (either positive or negative) to be applied to the current steering angle.
+- `ActionSpaceType.Lane`: `str` discrete lane action space of strings including "keep_lane",  "slow_down", "change_lane_left", "change_lane_right".
+- `ActionSpaceType.LaneWithContinuousSpeed`: `(int, float)` mixed action space containing discrete lane change `{-1,0,1}` and continuous target speed.
+- `ActionSpaceType.Trajectory`: `(Sequence[float], Sequence[float], Sequence[float], Sequence[float])` continuous action space with x coordinates, y coordinates, headings, and speeds.
+- `ActionSpaceType.TrajectoryWithTime`: `(Sequence[float], Sequence[float], Sequence[float], Sequence[float], Sequence[float])` continuous action space with x coordinates, y coordinates, headings, and speeds.
+- `ActionSpaceType.MPC`: `(Sequence[float], Sequence[float], Sequence[float], Sequence[float])` continuous action space with trajectory with times, x coordinates, y coordinates, headings, and speeds. 
+- `ActionSpaceType.MultiTargetPose`: `Dict[str, (float, float, float float)]` continuous action space with vehicle id mapped to x coordinate, y coordinate, heading, and future time of pose. 
+- `ActionSpaceType.Imitation`: `Union[float, (float,float)]` Case 1: continuous action space with speed. Case 2: Continuous action space with linear acceleration and angular velocity.
```

Comment:
I suggest the following:
- `ActionSpaceType.Imitation`: `Union[float, (float,float)]` continuous action space where you can pass either (a) initial speed upon reset or (b) linear acceleration and angular velocity for other steps.